bundles/org.openhab.ui/web/src/assets/definitions/blockly/blocks-notifications.js

```diff
@@ -5,9 +5,12 @@
 
 import Blockly from 'blockly'
 import { javascriptGenerator } from 'blockly/javascript.js'
+import { blockGetCheckedInputType } from './utils.js'
+import { underline } from 'chalk'
```

Comment:
Please remove this, I wonder how it got there:

```suggestion
```

Reply:
that's totally weird - indeed.